README.md

```diff
@@ -14,6 +14,75 @@ Thumbprint Tokens are published as JavaScript, SCSS, Swift, and Kotlin.
 
 You can use the [Thumbprint Tokens GraphQL server](https://thumbprint-tokens.netlify.com/) to query tokens and view the schema.
 
+Button, Theme, Title Color, Active, primary Tokens should follow this recommended hierarchy: `[<component-name>] <category-name> [<option-name> <state>] <token>`.
+
+-   For example, a simple token category like Space would look like:
+    ```
+    Space (Category)
+      |
+      |--- one (Token)
+      |--- two
+      ...
+    ```
+-   A more complex, component-level token category such as button themes might have a structure like:
+    ```
+    Button (Component)
+      |
+      |--- Theme (Category)
+             |
+             |--- Title Color (Option)
```

Comment:
What's the best way to ensure the names make sense on all platforms. I imagine `Title` from iOS? (For web we'd just call it text or font.)